portal-web/test/functional/com/liferay/portalweb/macros/ChangeList.macro

```diff
@@ -28,6 +14,32 @@ definition {
 		AssertConfirm.assertPartialConfirmation(value1 = "Do you want to switch to ${changeListName} change list?");
 	}
 
+	macro createNewPublication {
+		if (isSet(gotoManagePublications)) {
```

Comment:
To be more explicit, we should make sure that this is set to `true`, not just set